fern/docs/pages/docs/writing-content.mdx

```diff
@@ -223,6 +223,8 @@ tabs:
 ```
 </CodeBlock>
 
+![Let's replace this with the same img hosted on Fern's AWS S3 instead of Imgur](https://imgur.com/a/zQGqQWv)
```

Comment:
@dannysheridan I'd like to move this img into Fern's AWS S3 instance. Can you send me back a URL of this image hosted on Fern's AWS?